docs/components/modeler/about-modeler.md

```diff
@@ -35,3 +35,5 @@ In this guide, we'll demonstrate modeling BPMN diagrams using both Web Modeler a
 - [Modeling BPMN](/guides/automating-a-process-using-bpmn.md) - Learn how to quickly model an automated process using BPMN.
 - [Camunda Forms](/guides/utilizing-forms.md) - Allows you to easily design and configure forms. Once configured, they can be connected to a user task or start event to implement a task form in your application.
 - [DMN](./dmn/dmn.md) - In DMN, decisions can be modeled and executed using the same language. Business analysts can model the rules that lead to a decision in easy to read tables, and those tables can be executed directly by a decision engine (like Camunda).
+- [Out-of-the-box Connectors](/guides/implementing-connectors.md) - Integrate applications and services quickly to automate across systems using out-of-the-box Connectors.
```

Comment:
Minor tweak:
Update text from "Integrate applications and services quickly to automate across systems using out-of-the-box Connectors."
to "Out-of-the-box Connectors provide prebuilt connectivity with a number of outside systems to accelerate automation across those systems.